java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -852,6 +853,49 @@ private AddressInfo startObjectManager(int index, AddressInfo info, String workD
     }
   }
 
+  private Map<String, Double> getResourcesFromString(String resources) {
+    Map<String, Double> ret = new HashMap<>();
+    if (resources != null) {
+      String[] items = resources.split(",");
+      for (String item : items) {
+        String trimItem = item.trim();
+        String[] resourcePair = trimItem.split(":");
+
+        if (resourcePair.length != 2) {
+          RayLog.core.error("Static resources configure format error.");
+        }
+
+        String resourceName = resourcePair[0].trim();
+        if ("cpu".equals(resourceName)) {
+          resourceName = "CPU";
+        }
+
+        if ("gpu".equals(resourceName)) {
+          resourceName = "GPU";
+        }
```

Comment:
It may be preferable for it to be case sensitive. I don't feel too strongly about this, but I think it's generally nice to have one correct way of doing things.